zone/client_packet.cpp

```diff
@@ -8959,6 +8959,14 @@ void Client::Handle_OP_ItemVerifyRequest(const EQApplicationPacket *app)
 				}
 				if (GetLevel() >= item->Click.Level2)
 				{
+					if (item->RecastDelay > 0)
+					{
+						if (!GetPTimers().Expired(&database, (pTimerItemStart + item->RecastType), false)) {
+							LogSpells("Casting of [{}] canceled: item spell reuse timer not expired", spell_id);
```

Comment:
Do we not want a message here similar to the others?

Reply:
Client gives the message for regular item clicks. Would just be a double message. For clicks from augments it is needed. Good question though.

Reply:
Wasn't aware, wanted to make sure. 👍